_posts/2018-07-18-14-06-webapi-7-1-0.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title : "14.6 OData Web API 7.1.0 (.NET Core and .NET Classic)"
+description: "7.1.0 release notes"
+category: "14. 7.x Features "
+---
+
+The NuGet packages for ASP.NET Web API OData v7.1.0 are available on the [NuGet gallery](https://www.nuget.org/).
+
+You can install or update the NuGet packages for OData Web API v7.1.0 using the [Package Manager Console](http://docs.nuget.org/docs/start-here/using-the-package-manager-console):
+
+```
+PM> Install-Package Microsoft.AspNetCore.OData -Version 7.1.0 
+```
+or
+```
+PM> Install-Package Microsoft.AspNet.OData -Version 7.1.0
+```
+
+## What’s in this release?
+
+### New Features:
+
+* [ [Web API issue #1631](https://github.com/OData/WebApi/issues/1631) ] Singleton instance not possible without Keys
+
+* [ [Web API issue #1628](https://github.com/OData/WebApi/pull/1628) ] Allow adding new members to a collection through a POST request
+
+* [ [Web API issue #1591](https://github.com/OData/WebApi/issues/1591) ] OData builder does not honor namespace
```

Comment:
>OData builder does not honor namespace [](start = 73, length = 38)

Since this is listed under "New Features", restate as "Support namespaces in OData ModelBuilder"

Reply:
Updated. Thanks.